rflx/model/message.py

```diff
@@ -1569,6 +1569,52 @@ def proven(self, skip_proof: bool = False) -> "DerivedMessage":
         return copy(self)
 
 
+def replace_message_attributes(
+    message: AbstractMessage,
+) -> Tuple[AbstractMessage, List[Optional[Location]]]:
```

Comment:
This could be a method of `AbstractMessage` similar as `prefixed`. A method would be more consistent to the existing code.

Reply:
Changed in 877fd78d.